.travis.yml

```
@@ -21,7 +21,7 @@ script:
 - npm run build
 - npm run test-travis || travis_terminate 1
 - npm run lint
-- npm run check-packages
+- if [[ $(node --version) =~ v1[^0][\d.]* ]]; then npm run check-packages; fi
```

Nice fix!
I didn't run this suggestion locally, but for the people less versed in regex such as myself, this might be more natural / readable:
```suggestion
- if [[ $(node -e "process.exit(process.versions.node.split('.')[0] > 10);") ]]; then npm run check-packages; fi
```